### PR TITLE
Fixing error

### PR DESCRIPTION
**Pull request by gustavanderssonmind**
Merge fix to master

### CODE REVIEW

File: File1 [hunk 1]
```diff
@@ -5,7 +5,7 @@
 4
 5
 6
-8
+78
```
Comment (gustavanderssonmind) [inline File1]: I think this is wrong

File: File1 [hunk 2]
```diff
@@ -5,7 +5,7 @@
 4
 5
 6
-8
+7
```
Comment (gustavanderssonmind) [inline File1]: Much better now!

Comment (stez-mind) [inline File1]: Yes that looks great :+1:

**Comment by stez-mind** [comment]
This should fix #2

**Pull request merged**